### PR TITLE
enable linter & make deterministic

### PR DESCRIPTION
**Pull request by zramsay**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,11 +71,11 @@ get_vendor_deps: ensure_tools
 
 metalinter:
 	protoc $(INCLUDE) --lint_out=. types/*.proto
-	gometalinter --vendor --deadline=600s --enable-all --disable=lll ./...
+	gometalinter.v2 --vendor --deadline=600s --enable-all --disable=lll ./...
```
Comment (melekes) [inline Makefile]: we don't need the same `$(GOPATH)/bin/` here?

Comment (zramsay) [reply Makefile]: yea, not sure why. Suspect it has to do with the circle setup

File: Makefile [hunk 2]
```diff
@@ -71,11 +71,11 @@ get_vendor_deps: ensure_tools
...
 
 metalinter_test:
-	protoc $(INCLUDE) --lint_out=. types/*.proto
-	gometalinter --vendor --deadline=600s --disable-all  \
+	#protoc $(INCLUDE) --lint_out=. types/*.proto
```
Comment (melekes) [inline Makefile]: either remove or uncomment

File: Makefile (hunk 1 above)
Comment (melekes) [reply Makefile]: ok

**Pull request merged**